src/main/java/edu/cornell/kfs/sys/service/impl/ReportWriterTextServiceImpl.java

```diff
@@ -104,67 +101,56 @@ public String getFullFilePath() {
 		return fullFilePath;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public String getFromAddress() {
 		return fromAddress;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public void setFromAddress(String fromAddress) {
 		this.fromAddress = fromAddress;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public Set<String> getCcAddresses() {
 		return ccAddresses;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public void setCcAddresses(Set<String> ccAddressList) {
 		this.ccAddresses = ccAddressList;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public String getMessageBody() {
 		return messageBody;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public void setMessageBody(String messageBody) {
 		this.messageBody = messageBody;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public RoleService getRoleService() {
 		return roleService;
 	}
 
-	/**
-	 *
-	 */
 	@Override
 	public void setRoleService(RoleService roleService) {
 		this.roleService = roleService;
 	}
+	
+	@Override
+	public void writeFormattedMessageLine(String format, Object... args) {
+		super.writeFormattedMessageLine(format, args);
+		if (LOG.isDebugEnabled()) {
```

Comment:
this code could be extracted in a separate method that just does the logging and is called here after the super call.